resources/v2/01_drivers.sql

```diff
@@ -0,0 +1,14 @@
+CREATE TABLE drivers (
+    id INTEGER NOT NULL PRIMARY KEY,
```

Comment:
Make it an identity column https://www.postgresqltutorial.com/postgresql-tutorial/postgresql-identity-column/

Reply:
done